webapp/tests/test_dashboard.py

```diff
@@ -1,9 +1,226 @@
+import copy
+import json
+
+from django.conf import settings
+
 from django.core.urlresolvers import reverse
 from django.test import TestCase
 
-
 class DashboardTest(TestCase):
     def test_dashboard(self):
+
```

Comment:
Generally speaking, I think it is better practice to keep test methods short, with explicit names on what they do. There are 50 assertions in this one test method! This implies that laters assertions might be dependant on the state left by previous test assertions − if that is the case, it should be done as part of a `setUp` method.

The comments left actually make good basis for methods like `test_append_dashboard_name_to_url_returns_200()`